gui/app/periodic_graph.h

```diff
@@ -0,0 +1,124 @@
+#ifndef CIRCULARLINESERIES_H
+#define CIRCULARLINESERIES_H
+
+#include "periodic_graph_painter.h"
+#include "qnanoquickitem.h"
+#include "qnanoquickitempainter.h"
+#include <QColor>
+#include <QPointF>
+#include <QQuickItem>
+#include <QVector>
+
+/**
+ * @brief The PeriodicGraph is an QuickItem used to display a periodic series.
```

Comment:
Same here - I think "periodic" is the wrong term but I'm not sure yet what you mean.